src/io/flutter/dart/DartSyntax.java

```diff
@@ -70,21 +70,20 @@ public static DartCallExpression findEnclosingFunctionCall(@NotNull PsiElement e
    *
    * @return true if the given element is a call to function, false otherwise
    */
-  public static boolean isCallToFunctionNamed(@Nullable PsiElement element, @NotNull String function) {
-    if (!(element instanceof DartCallExpression)) return false;
-    final String name = getCalledFunctionName((DartCallExpression)element);
+  public static boolean isCallToFunctionNamed(@NotNull DartCallExpression element, @NotNull String function) {
```

Comment:
No, but there should be more tests for sure.

Also, interestingly, I've got classpath issues and can't run these tests in EAP; will look into fixing that.